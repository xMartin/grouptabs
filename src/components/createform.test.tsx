import renderer from "react-test-renderer";
import CreateForm from "./createform";

it("renders form", () => {
  const tree = renderer
    .create(
      <CreateForm
        tabName="Badminton"
        onTabNameChange={vi.fn()}
        onSubmit={vi.fn()}
      />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
