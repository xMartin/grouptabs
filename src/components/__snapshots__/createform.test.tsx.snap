// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders form 1`] = `
<form
  className="create-form"
  onSubmit={[Function]}
>
  <input
    className="full-width"
    onChange={[Function]}
    placeholder="Group title â€¦"
    type="text"
    value="Badminton"
  />
  <button
    className="create"
  >
    Create
  </button>
</form>
`;
